port/nordic/nrf5x/src/hal/gpio.zig

```diff
@@ -0,0 +1,85 @@
+const microzig = @import("microzig");
+const peripherals = microzig.chip.peripherals;
+const P0 = peripherals.P0;
+const GPIOTE = microzig.GPIOTE;
+
+pub const Direction = enum(u1) {
+    input = 0x0,
+    output = 0x1,
+};
+
+pub const Sense = enum(u2) {
+    disabled = 0x0,
+    /// sense high level
+    high = 0x2,
+    /// sense low level
+    low = 0x3,
+    _,
+};
+
+pub const Drive = @FieldType(P0.PIN_CNF[0].underlaying_type, "DRIVE");
```

Comment:
typo with underlying type, but also, I never got this working. I had to expose the type via a patch.

If we want this to look like rp2xxx we might want to call it DriveStrength... but then again, it's not really the same.